retail/interactive-tutorial/RetailSearch.Samples/SearchWithFacetSpecSample.cs

```diff
@@ -0,0 +1,103 @@
+
+using Google.Cloud.Retail.V2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+/// <summary>
+/// Search with facet spec sample class.
+/// </summary>
+public class SearchWithFacetSpecSample
+{
+    /// <summary>Get search request.</summary>
+    private SearchRequest GetSearchRequest(string query, string facetKeyParam, string projectNumber)
+    {
+        string defaultSearchPlacement = $"projects/{projectNumber}/locations/global/catalogs/default_catalog/placements/default_search";
+
+        // Put the intervals here:
+        var searchRequest = new SearchRequest()
+        {
+            Placement = defaultSearchPlacement, // Placement is used to identify the Serving Config name
+            Query = query,
+            VisitorId = "123456", // A unique identifier to track visitors
+            PageSize = 10,
+            FacetSpecs =
+            {
+                new SearchRequest.Types.FacetSpec
+                {
+                    FacetKey = new SearchRequest.Types.FacetSpec.Types.FacetKey
+                    {
+                         Key = facetKeyParam
+                    }
+                }
+            }
+        };
+
+        Console.WriteLine("\nSearch. request:\n");
+        Console.WriteLine($"Placement: {searchRequest.Placement}");
+        Console.WriteLine($"Query: {searchRequest.Query}");
+        Console.WriteLine($"VisitorId: {searchRequest.VisitorId}");
+        Console.WriteLine($"PageSize: {searchRequest.PageSize}");
+        Console.WriteLine($"FacetSpecs: {searchRequest.FacetSpecs}");
+
+        return searchRequest;
+    }
+
+    /// <summary>Call the Retail Search.</summary>
+    [RetailSearch.Samples.Attributes.Example]
+    public IEnumerable<SearchResponse> Search(string projectNumber)
+    {
+        // Try different facets here:
+        string facetKey = "colorFamilies";
+        string query = "Tee";
+
+        var client = SearchServiceClient.Create();
+        var searchRequest = GetSearchRequest(query, facetKey, projectNumber);
+        var searchResponses = client.Search(searchRequest).AsRawResponses();
+
+        var firstSearchResponse = searchResponses.FirstOrDefault();
+
+        Console.WriteLine("\nSearch. response: \n");
+
+        if (firstSearchResponse != null)
+        {
+            Console.WriteLine($"Results: {firstSearchResponse.Results}");
+            Console.WriteLine($"TotalSize: {firstSearchResponse.TotalSize},");
+            Console.WriteLine($"AttributionToken: {firstSearchResponse.AttributionToken},");
+            Console.WriteLine($"NextPageToken: {firstSearchResponse.NextPageToken},");
+            Console.WriteLine($"Facets: {firstSearchResponse.Facets},");
+        }
+
+        return searchResponses;
+    }
+}
+// [END retail_search_product_with_facet_spec]
+
+/// <summary>
+/// Search with facet spec tutorial.
+/// </summary>
+public static class SearchWithFacetSpecTutorial
+{
+    [RetailSearch.Samples.Attributes.Example]
```

Comment:
Ah, I see you have it here, you can just remove it from within the sample.

Reply:
Removed the Attribute from the Sample class.